### PR TITLE
[[ Documentation ]] Fixes to frontScripts.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
- Added messages tag.
- Added script (glossary) and message (glossary) to references.
- Improved example.
- Added links to referenced tokens.

### CODE REVIEW

File: docs/dictionary/function/frontScripts.lcdoc [hunk 1]
```diff
@@ -27,28 +27,29 @@ The <frontScripts> function returns a list of the long ID property of
...
+first <object> in the <frontScripts> receives <message|messages> before 
+the second <object>. This order is the reverse of the order in which the 
+<object|objects> were added with the <insert script> command.
 
 References: remove script (command), insert script (command),
-send (command), pass (control structure), object (glossary),
-message path (glossary), return (glossary)
+message (glossary), pass (control structure), object (glossary),
+message path (glossary), return (glossary), send (command), 
+script (glossary)
```
Comment (montegoulding) [inline docs/dictionary/function/frontScripts.lcdoc]: @asayd same commend as with backScripts... also perhaps dispatch and call should be here???

**Comment by montegoulding** [comment]
👍 
@livecode-vulcan review ok b80638b

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok b80638b2ee401294843920d840143430ef316df5

**Comment by livecode-vulcan** [comment]
😎 test success b80638b2ee401294843920d840143430ef316df5
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/95)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/95)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/95)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/95)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/95)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/95)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/95)

**Pull request merged**